fastlane/lanes/build.rb

```diff
@@ -68,6 +68,8 @@
 
     UI.user_error!("Unable to find .xctestrun file at #{build_products_path}") if test_plan_path.nil? || !File.exist?((test_plan_path))
 
+    add_buildkite_analytics_token(xctestrun_path: test_plan_path)
```

Comment:
`test_plan_path` is not the right name for the variable. https://github.com/wordpress-mobile/WordPress-iOS/pull/18922 fixes it.